### PR TITLE
Jd/add swipe

### PR DESCRIPTION
**Pull request by dittonjs**
This PR adds swipe functionality to the homescreen for deleting and sharing, see #96. 

Android
iOS

### CODE REVIEW

**Comment by bmuramatsu** [comment]
Could you please ask Brandon Findlay to recommend a color for the background on the share side. Perhaps a different gray?

For the delete, can he recommend a color? I'm thinking the same color red as Apple Mail and Notes. It might be SwiftUI systemRed (see (https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/))

I don't use Android to know what the "typical" delete background color is. I presume Google specifies it somewhere.

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
Looks good. We'll track the request for changing of color for delete in a separate issue/PR.

File: ios/Podfile.lock [hunk 1]
```diff
@@ -238,7 +238,7 @@ PODS:
   - React-jsinspector (0.63.4)
   - react-native-aes-cipher (1.0.9):
     - React
-  - react-native-app-auth (6.4.2):
+  - react-native-app-auth (6.4.1):
```
Comment (dmitrizagidulin) [inline ios/Podfile.lock:241]: I wonder why this package (and the next two) got downgraded, here?

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
I'm going to undo the changes to `Podfile.lock`, since I suspect those were added accidentally, and merge.

**Pull request merged**